tests/step-108.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+# Test:
+#   Custom installation prefix test
+
+INSTALL_DIR="/tmp/githooks"
```

Comment:
I would avoid setting this into the same variable (or any variable to be honest) here to make sure the code doesn't actually only work because the test sets the variable that the code will also expect later.

Reply:
Lol, jeah :-)